### ISSUE TITLE
Detect use of "url:file://"

### ISSUE THREAD

**Issue by RedXanadu** [coreruleset/coreruleset#2726]
## Description

Relates to **YJGXRICC**

Associated PR: #2727

---

We catch things like 

```
GET /x?x=file://etc/services HTTP/1.1

->

931130 PL2 Possible Remote File Inclusion (RFI) Attack: Off-Domain Reference/Link
```

However: "in Java applications using `org.apache.commons.io.IOUtils`, one can use":

```
url:file://etc/services
```

which we do not catch.

---

- [x] Extend rule 931130 to catch `url:file`
- [x] Add an associated test

**Comment by RedXanadu** [comment]
PR Merged. Closing.
